packages/docs-reanimated/src/components/InteractivePlayground/useEnteringExitingAnimationPlayground/Example.tsx

```diff
@@ -0,0 +1,270 @@
+  BounceOut: BounceOut,
+  BounceOutRight: BounceOutRight,
+  BounceOutLeft: BounceOutLeft,
+  BounceOutUp: BounceOutUp,
+  BounceOutDown: BounceOutDown,
+
+  // Flip
+  FlipOutEasyX: FlipOutEasyX,
+  FlipOutEasyY: FlipOutEasyY,
+  FlipOutXDown: FlipOutXDown,
+  FlipOutXUp: FlipOutXUp,
+  FlipOutYLeft: FlipOutYLeft,
+  FlipOutYRight: FlipOutYRight,
+
+  // LightSpeed
+  LightSpeedOutRight: LightSpeedOutRight,
+  LightSpeedOutLeft: LightSpeedOutLeft,
+
+  // Pinwheel
+  PinwheelOut: PinwheelOut,
+
+  // Roll
+  RollOutRight: RollOutRight,
+  RollOutLeft: RollOutLeft,
+
+  // Rotate
+  RotateOutDownLeft: RotateOutDownLeft,
+  RotateOutDownRight: RotateOutDownRight,
+  RotateOutUpLeft: RotateOutUpLeft,
+  RotateOutUpRight: RotateOutUpRight,
+
+  // Slide
+  SlideOutRight: SlideOutRight,
+  SlideOutLeft: SlideOutLeft,
+  SlideOutUp: SlideOutUp,
+  SlideOutDown: SlideOutDown,
+
+  // Stretch
+  StretchOutX: StretchOutX,
+  StretchOutY: StretchOutY,
+};
+
+export default function Example({ entering, exiting }: ExampleProps) {
+  const [visible, setVisible] = React.useState(true);
+
+  const getEnteringAnimation = () => {
+    const {
+      animation,
+      easing,
+      duration,
+      delay,
+      isSpringBased,
+      mass,
+      stiffness,
+      damping,
+      overshootClamping,
+      restSpeedThreshold,
+      restDisplacementThreshold,
+    } = entering;
+
+    return !isSpringBased
+      ? ENTERING_ANIMATIONS[animation]
+          .duration(duration)
+          .delay(delay)
+          .easing(easing)
+      : ENTERING_ANIMATIONS[animation]
+          .delay(delay)
+          .springify()
+          .damping(damping)
+          .mass(mass)
+          .overshootClamping(overshootClamping)
+          .restSpeedThreshold(restSpeedThreshold)
+          .restDisplacementThreshold(restDisplacementThreshold)
+          .stiffness(stiffness);
+  };
+
+  const getExitingAnimation = () => {
+    const {
+      animation,
+      easing,
+      delay,
+      duration,
+      isSpringBased,
+      mass,
+      stiffness,
+      damping,
+      overshootClamping,
+      restSpeedThreshold,
+      restDisplacementThreshold,
+    } = exiting;
+
+    return !isSpringBased
+      ? EXITING_ANIMATIONS[animation]
+          .duration(duration)
+          .delay(delay)
+          .easing(easing)
+      : EXITING_ANIMATIONS[animation]
+          .springify()
+          .delay(delay)
+          .damping(damping)
+          .mass(mass)
+          .overshootClamping(overshootClamping)
+          .restSpeedThreshold(restSpeedThreshold)
+          .restDisplacementThreshold(restDisplacementThreshold)
+          .stiffness(stiffness);
+  };
+
+  return (
+    <View style={styles.container}>
+      <Button title="Toggle View" onPress={() => setVisible((prev) => !prev)} />
+      {visible && (
+        <Animated.View
+          entering={getEnteringAnimation()}
+          exiting={getExitingAnimation()}
+          style={styles.box}></Animated.View>
+      )}
```

Comment:
I know some features are not fully implemented on web, but how about the option to add a child to this component?

Reply:
(in off screen discussion we've decided to doesn't add child)

Reply:
❌ 🧒 ❌